src/main/webapp/assets/modules/freelancer/post-project/postProjectController.js

```diff
@@ -44,6 +44,21 @@ techlooper.controller('freelancerPostProjectController', function ($scope, jsonV
         var estimatedEndDate = moment($scope.fixedPrice.estimatedEndDate, jsonValue.dateFormat);
         if (!estimatedEndDate.isValid()) return false;
         return (estimatedEndDate.isAfter(moment(), 'day') || estimatedEndDate.isSame(moment(), "day"));
+
+      case "hourly-rate-gt-0":
+        if (!$scope.hourly) return true;
+        if ($scope.hourly.hourlyRate == undefined) return true;
```

Comment:
Use '===' to compare with 'undefined'.<br>Expected '{' and instead saw 'return'.